### PR TITLE
A start on RPMS

### PR DESCRIPTION
**Pull request by nichenke**
This is the basic plumbing and infra needed to get some ".rpm" files emitted after Travis runs a thing. 

The RPMs are not yet deployable for python-libarchive, as we need some code tweaks to make xattr support possible.  We'll add in Artifactory upload once local testing is happy and we push those RPM updates into future PRs here.

### CODE REVIEW

File: .versity/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM centos:7
+
+MAINTAINER Nic Henke <nic.henke@versity.com>
+
+# Packages needed to build libarchive from source, including a few utilities to help with debugging
+RUN yum install -y epel-release && \
```
Comment (gcymbalski) [inline .versity/Dockerfile:6]: Hrrrrm, should we include these in the docker-rpm-build definitions and let this project use that image?

**Review by heemstra — COMMENTED** [review]
Looks good... if we had some existing administrator documentation I'd ask if we should be adding verbiage about installing a package in a non-standard location that shadows something that could potentially be installed elsewhere (libarchive in this case.)  But we have no such documentation... so this is more of a thought bubble I guess.  Seems kinda irrelevant in this particular case, but it triggered the thought nonetheless.

**Review by gcymbalski — CHANGES_REQUESTED** [review]

File: .versity/Dockerfile (hunk 1 above)
Comment (nichenke) [reply .versity/Dockerfile:6]: We totally could, I didn't want to hold this up with a cycle through that project until we had something workable. It seems like a good next step. Do you mind if I clean that up in subsequent commits to allow me to get the artifact deployment worked out first ?

Comment (gcymbalski) [reply .versity/Dockerfile:6]: Sounds great, just wasn't sure if there were too many specifics to make it worthwhile 👍

**Review by gcymbalski — APPROVED** [review]

**Pull request merged**